### PR TITLE
Update to 5.15.6

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by hmaarrfk** [comment]
Is this ready?

**Comment by hmaarrfk** [comment]
If so, do you mind adding this file: https://github.com/conda-forge/qt-main-feedstock/pull/47/files#diff-639f9b797377cbe9a41206de59731c03afb55563f0063ea1c678786f4dce91b6

**Review by Tobias-Fischer — APPROVED** [review]

**Comment by hmaarrfk** [comment]
@Tobias-Fischer are you able to build aarch?

**Comment by Tobias-Fischer** [comment]
Not at the moment - the disk is full on the conda-forge aarch server:
```
/dev/sda3        39G   39G   20K 100% /
```

**Comment by isuruf** [comment]
Cleaned it up for you.
```
/dev/sda3        39G   13G   26G  33% /
```

File: .scripts/run_osx_build.sh [hunk 1]
```diff
@@ -25,9 +25,9 @@ conda activate base
 
 echo -e "\n\nInstalling ['conda-forge-ci-setup=3'] and conda-build."
 mamba install --update-specs --quiet --yes --channel conda-forge \
-    conda-build pip boa conda-forge-ci-setup=3
+    conda-build liblief=0.11.5 pip boa conda-forge-ci-setup=3
 mamba update --update-specs --yes --quiet --channel conda-forge \
-    conda-build pip boa conda-forge-ci-setup=3
+    conda-build liblief=0.11.5 pip boa conda-forge-ci-setup=3
```
Comment (hmaarrfk) [inline .scripts/run_osx_build.sh:30]: Can you add:

```
remote_ci_setup:
- liblief=0.11.5
```
to conda-forge.yml

https://github.com/conda-forge/conda-smithy/pull/1579/files#diff-d889cd3c5e3aa36e3c852ab58fe7f8abae463e760186f7cae7b40d93646852aeR4

File: .ci_support/osx_64_.yaml [hunk 2]
```diff
@@ -5,15 +5,15 @@ MACOSX_SDK_VERSION:
...
+- '13'
 channel_sources:
 - conda-forge
 channel_targets:
 - conda-forge main
 cxx_compiler:
 - clangxx
 cxx_compiler_version:
-- '14'
+- '13'
```
Comment (hmaarrfk) [inline .ci_support/osx_64_.yaml:16]: is it possible to add this to a configuration file instead?

**Comment by Tobias-Fischer** [comment]
Should we integrate https://github.com/conda-forge/qt-main-feedstock/pull/45 here?

**Comment by Tobias-Fischer** [comment]
linux-aarch64 build: https://anaconda.org/TobiasRobotics/qt-main/5.15.5/download/linux-aarch64/qt-main-5.15.5-hb0efc4c_0.tar.bz2

log: https://www.dropbox.com/s/cwqndn4axvif04n/qt-aarch-build.log?dl=0

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin please rerender

**Review by andfoy — COMMENTED** [review]
I think that the build log got push'd into the diff

**Comment by Tobias-Fischer** [comment]
@andfoy - I am not sure what you mean?

**Comment by andfoy** [comment]
See https://github.com/conda-forge/qt-main-feedstock/blob/c023bc7f622415faf661fcb9e8ed5a4c2b55bc44/recipe/build.log

**Comment by Tobias-Fischer** [comment]
Thanks for the catch!

**Comment by hmaarrfk** [comment]
can you force push instead?

**Comment by hmaarrfk** [comment]
to delete it from the history

**Comment by Tobias-Fischer** [comment]
Done

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
@conda-forge-admin please rerender

**Comment by jschueller** [comment]
I bumped to 5.15.6 while at it

**Comment by Tobias-Fischer** [comment]
Awesome, thanks for that @jschueller! My question from above is still open - should we integrate https://github.com/conda-forge/qt-main-feedstock/pull/45? I think it would be beneficial. If we agree it is, I can rebuild on aarch64.

**Comment by jschueller** [comment]
Ok, done

**Comment by Tobias-Fischer** [comment]
Updated package: https://anaconda.org/TobiasRobotics/qt-main/5.15.6/download/linux-aarch64/qt-main-5.15.6-hb0efc4c_0.tar.bz2
Updated build log: https://www.dropbox.com/s/hsspbzqxqggq159/qt-aarch-build5156.log?dl=0

**Pull request merged**